Foundation/NSData.swift

```diff
@@ -930,7 +930,7 @@ open class NSMutableData : NSData {
     }
     
     open func resetBytes(in range: NSRange) {
-        bzero(mutableBytes.advanced(by: range.location), range.length)
+        memset(mutableBytes.advanced(by: range.location), 0, range.length)
```

Comment:
What was wrong with `bzero` here? I think I would rather have a compatibility shim for Android than excise it from our code, if it's missing.